webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js

```diff
@@ -217,3 +217,29 @@ describe('Without tracer installed', () => {
     expect(enableTracesModalLink).toHaveClass('pf-m-in-progress');
   });
 });
+
+test('verify banner is present for old host page', async (done) => {
+  // Setup autocomplete with mockForemanAutoComplete since we aren't adding /katello
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+
+  const noResults = {
+    total: 0,
+    subtotal: 0,
+    page: 1,
+    per_page: 20,
+    results: [],
+  };
+
+  const scope = nockInstance
+    .get(hostTraces)
+    .reply(200, noResults);
+
+  const { queryByText } = renderWithRedux(<TracesTab />, renderOptions);
+
+  // Assert that there are not any traces showing on the screen.
+  await patientlyWaitFor(() => expect(queryByText('Traces functionality on this page is incomplete.')).toBeInTheDocument());
+  await patientlyWaitFor(() => expect(queryByText('Visit the previous Traces page.')).toBeInTheDocument());
```

Comment:
I don't think this one needs an `await patientlyWaitFor`
```suggestion
  expect(queryByText('Visit the previous Traces page.')).toBeInTheDocument();
```